src/common/i18n/translations/en.ts

```diff
@@ -398,5 +398,10 @@ export const en: Translations = {
       body: "Enter valid email address",
       primaryActionText: "OK",
     },
+    invalidDeviceCode: {
+      title: "Invalid Input",
+      body: "Scan item again or get a new item from your in-charge.",
+      primaryActionText: "Ok",
```

Comment:
Just some minor typography changes:

```suggestion
      title: "Invalid input",
      body: "Scan item again or get a new item from your in-charge.",
      primaryActionText: "OK",
```